main/__tests__/IconButton.test.tsx

```diff
@@ -123,11 +116,10 @@ describe('[IconButton]', () => {
       expect(json).toBeTruthy();
     });
 
-    it('should render type==="secondary"', () => {
+    it('should render color==="secondary"', () => {
```

Comment:
~~should render color==="secondary"~~ => renders secondary color

would be more concise